docs/semgrep-appsec-platform/jira.md

```diff
@@ -142,7 +151,7 @@ To create tickets:
 1. Set the status to **Open**, **Fixing**, or **Reviewing**. Select **Fixing** if it is a known issue that needs to be fixed or **Reviewing** if the finding needs more investigation.
 1. Select the **Create tickets...** checkbox.
     1. Optional: Click the first drop-down list to choose between making a ticket for **groups of findings** or an individual ticket for **each finding**.
-    1. Optional: Click the **JIRA project** drop-down list to select which Jira project to add the findings to.
+    1. Optional: Click the **JIRA project** drop-down list to select which Jira project to add the findings to. You can choose any project that is associated with the issue type configured in your integration settings.
```

Comment:
Not related to components but think this would be good to mention here

Reply:
SGTM!